washer-start/public/index.html

```diff
@@ -45,6 +45,20 @@ <h3 class='user-welcome-heading' id='user-welcome'></h3>
     </header>
     <main class="mdl-layout__content mdl-color--grey-100">
       <div class="mdl-cell--16-col mdl-cell--16-col-tablet mdl-grid">
+
+        <!-- Card containing the sign-in UI -->
```

Comment:
We can get rid of the signed-out-card since this codelab doesn't do any client-side auth.

Reply:
@Fleker Ok, I see. I thought that we have following two options to fix the issue. I understand that you want to apply the 2nd option below.

* Add the element set for the "Card containing the sign-in UI".
* Remove the line which refers the element at [washer-start/public/main.js#L56](https://github.com/googlecodelabs/smarthome-washer/blob/master/washer-start/public/main.js#L56).

I intend to delete the 5699d0b Then I will commit the code to achieve the 2nd option above.

Reply:
Yes, there's not much reason to have sign-in. Thanks.